.github/workflows/ci.yml

```diff
@@ -86,12 +86,13 @@ jobs:
     - name: Test
       uses: actions-rs/cargo@v1
       env:
+        CONN_STRING: https://127.0.0.1:8091
```

Comment:
If this is only running unit tests then why does it need CONN_STRING?

Reply:
An artefact from trying to get things passing, but yes we don't need it.